avaframe/runQuickPlot.py

```diff
@@ -24,11 +24,14 @@
 log.info('Current avalanche: %s', avalancheDir)
 
 # REQUIRED+++++++++++++++++++
+# Which parameter to filter data, e.g. varPar = 'simType', values = ['null'] or varPar = 'Mu', values = ['0.055', '0.155']
+ # values need to be given as list, also if only one value
 outputVariable = ['pfd', 'ppr']
-simName = 'entres'
+values = ['null']
+varPar = 'simType'
```

Comment:
Would it make sense to include these in a ini file as default values?

Reply:
I added a outQuickPlotCfg.ini file with default values, and now the values and parameter used for filtering the data can be set there